android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/app/ListSyncConfig.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021-2024 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.configuration.app
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ListSyncConfig(
+  val page: Int = 1,
+  val count: Int = 20,
+)
```

Comment:
This new configuration can be hardcoded to a reasonable number unless there is a use case for the flexibility.

Reply:
@ellykits So I put it that way because those are the default values on the server side. What would be the recommended value for us on the client side?

Reply:
We can hardcode to 50 and test out how it behaves.In the end it will depend on the backend performance by we can start with this and maybe tweak upwards or downwards. 

Also based on my [previous comment here](https://github.com/opensrp/fhircore/pull/3125#discussion_r1522715292), we don't actually need to make pageNumber configurable. It can be hardcoded as well for the first fetch

Reply:
Oooh okay. So Lemme change the number to 50. Then I also hard code the pageNumber and page count as well. @ndegwamartin should I then remove the listSync from the app config completely?